addon/classes/Column.js

```diff
@@ -129,11 +130,37 @@ export default class Column extends EmberObject.extend({
 
   /**
    * The minimum width (in px) that this column can be resized to.
-   * @property minResizeWidth
+   * @property _minResizeWidth
    * @type {Number}
    * @default 0
+   * @private
+   */
+  _minResizeWidth: 0,
+
+  /**
+   * The minimum width that this column can be resized to.
+   * @property minResizeWidth
+   * @type {String}
+   * @default '0px'
    */
-  minResizeWidth: 0,
+  minResizeWidth: computed({
+    get() {
+      return this.get('_minResizeWidth');
+    },
+    set(key, value) {
+      if (typeOf(value) === 'string') {
+        let [, quantity, units] = value.match(/(\d+)(px)?/) || [];
+        if (units !== 'px') {
+          warn('`value` attribute is interpreted as px regardless of provided units', {
+            id: 'ember-light-table.classes.Column'
+          });
+        }
+        this.set('_minResizeWidth', Number(quantity));
+      } else {
+        this.set('_minResizeWidth', value);
```

Comment:
```js
return this.set('_minResizeWidth', value);
```

Also, what about other non-string values? What if I pass `NaN`, `{}`, `false`, `null`, ...? I think we should handle these cases a bit more explicitly and maybe even throw assertions / `TypeError`s for some scenarios.

Reply:
@buschtoens The only concern I have about throwing assertions for `TypeError`s is that it could cause people who are incorrectly using `minResizeWidth` currently to receive an exception which may negatively affect their application. Now you can argue that since they are already using it incorrectly, this is actually better, but it is something to think about. 

I plan on making the suggested changes this upcoming week.